### DISCUSSION TITLE
Direct name of nested routing

### DISCUSSION THREAD

**Discussion by Sandeep06Dev** [vercel/next.js#15447] [Help] [answered]
#Hi..
I have a page  "car_service.jsx" inside "pages/services/car_services.Jsx" 

Now i want to load this page by visiting "localhost:3000/Car".
How can i achieved it 

Help please..

**Comment by jamesmosier** [comment, marked as answer]
There is an experimental (soon to be released to stable) feature for rewrites and redirects. If you install the latest canary (`9.4.5-canary.43`) you can access it like this below:

`next.config.js`

```js
module.exports = {
    async redirects() {
      return [
        {
          source: '/Car',
          destination: '/services/car_services',
          permanent: true
        }
      ]
    }
}
```

or with rewrites

```js
module.exports = {
  async rewrites() {
    return [
      {
        source: '/Car',
        destination: '/services/car_services',
      },
    ];
  },
};

```

Here is the [RFC](https://github.com/vercel/next.js/discussions/9081)
